Libs/RaptureCore/src/test/java/rapture/kernel/scripting/ScriptInterfaceTest.java

```diff
@@ -76,7 +80,17 @@ public void testInputProgram() throws RecognitionException {
         //assertTrue(testScript("println('hello');println(_params['hello']);return 5;\n").isEmpty());
     	
         ReflexRaptureScript rrs = new ReflexRaptureScript();
-        String test = getResourceAsString(this, "/scriptinterface.script");
+        String test = "";
```

Comment:
A lot of the test cases don't have any checking to make sure that they actually ran successfully. Unless there's a specific failure they quietly pass.